test/e2e/cypress/support/commands.ts

```diff
@@ -1269,20 +1266,20 @@ Cypress.Commands.add(
           cy.wait(pollInterval)
           return req()
         }
+
         let mailItem: any
         if (email) {
           mailItem = response.body.mailItems.find((m: any) =>
             m.toAddresses.includes(email),
           )
           if (!mailItem) {
-            return req
+            tries++
+            cy.wait(pollInterval)
+            return req()
```

Comment:
This caused flakes - because no waiting and req returned as a function, not a function call.